core/src/main/scala/io/odin/loggers/ConsoleLogger.scala

```diff
@@ -27,11 +27,12 @@ case class ConsoleLogger[F[_]](
     formatter: Formatter,
     out: PrintStream,
     err: PrintStream,
-    override val minLevel: Level
+    override val minLevel: Level,
+    syncType: Sync.Type = Sync.Type.Blocking
```

Comment:
Instead of defining a default value here, I think it's better to create a smart constructor in the companion object 👇🏽 

```Scala
  def apply[F[_]: Sync](formatter: Formatter, minLevel: Level): Logger[F] =
    apply(formatter, minLevel, Sync.Type.Blocking)

  def apply[F[_]: Sync](formatter: Formatter, minLevel: Level, syncType: Sync.Type): Logger[F] =
    ConsoleLogger(formatter, scala.Console.out, scala.Console.err, minLevel, syncType)
```

Reply:
For consistency with how it's done in both `FileLogger` and `Slf4jLogger`, I went with adding the new param as default to the existing `apply` rather than overloading. Overloading is more bincompat-friendly, but it doesn't matter for this case (bumping minor anyway) and IMHO, a switch should be done once for all and not only for new additions.

Reply:
Probably I should've given you more context. I need to investigate deeper but I'm evaluating if it's feasible to hide the concrete classes (`Slfj4Logger`, `ConsoleLogger`...) and expose only `Logger[F]`. That would be possible if the constructors are defined in the companion object. I want to try this now that bin-compat is already broken.